models/Notification/Service/UserService.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+declare(strict_types=1);
+
+namespace Pimcore\Model\Notification\Service;
+
+use Pimcore\Model\User\Role;
+use Pimcore\Model\User;
+
+class UserService
+{
+    /**
+     * @param User $user
+     * @return array
+     */
+    public function findAll(User $loggedIn): array
+    {
+        $users = [];
+        $roles = [];
+
+        $filter = [
+            'id > ?'     => 0,
+            'id != ?'    => $loggedIn->getId(),
+            'name != ?'  => 'system',
+            'active = ?' => 1,
+        ];
+
+        $condition          = implode(' AND ', array_keys($filter));
+        $conditionVariables = array_values($filter);
+
+        $listing = new User\Listing();
+        $listing->setCondition($condition, $conditionVariables);
```

Comment:
we should filter here for users that have the 'notification' permission ... otherwise you could send a notification to a user, that never sees it :)

Reply:
in addition to that, we need to exclude type 'userfolder' here ... otherwise user folders will appear also in this list